Submitted By: BLFS Book <blfs-book@linuxfromscratch.org>
Date: 2003-10-03
Initial Package Version: 5.50
Origin: NA
Description: Creates a missing symlink.

$LastChangedBy: randy $
$Date: 2005-03-26 16:31:35 -0700 (Sat, 26 Mar 2005) $

diff -urN unzip-5.50/unix/Makefile unzip-5.50-rcl/unix/Makefile
--- unzip-5.50/unix/Makefile	Sat Feb 16 17:00:38 2002
+++ unzip-5.50-rcl/unix/Makefile	Sat Sep 28 14:32:44 2002
@@ -818,6 +818,7 @@
 	ln -sf crc_gcc.pic.o crc32.pic.o
 	gcc -shared -Wl,-soname,libunzip.so.0 -o libunzip.so.0.4 $(OBJSDLL)
 	ln -sf libunzip.so.0.4 libunzip.so.0
+	ln -sf libunzip.so.0.4 libunzip.so
 	gcc -c -O unzipstb.c
 	gcc -o unzip unzipstb.o -L. -lunzip -lz
